tasks/definitions/cucumber/run.js

```diff
@@ -10,18 +10,33 @@ const path = require('path')
  *                          See options section of the task definition below
  */
 const buildCmdArgs = params => {
-  const { context: name, jestConfig, testDir, timeout } = params
+  const { jestConfig, timeout } = params
```

Comment:
Not a big deal, but seems like this task should be changed to `parkin` instead of `cucumber`

Reply:
Tried to, but keg-cli seems to be interfering.